Lib/test/libregrtest/result.py

```diff
@@ -0,0 +1,185 @@
+import dataclasses
+import json
+from typing import Any
+
+from . import TestStats
+from .utils import format_duration, normalize_test_name, print_warning
+
+
+# --match and --ignore options: list of patterns
+# ('*' joker character can be used)
+FilterTuple = tuple[str]
+
+
+# Avoid enum.Enum to reduce the number of imports when tests are run
```

Comment:
Now when you import `dataclasses` you import half of the stdlib: it imports `inspect` and it imports `enum`.

Maybe this can be made into an `StrEnum` as it should be?

Reply:
Oh no. That's bad :-( I should maybe rewrite the code to avoid this heavy ``dataclasses`` module. I didn't know that it has so many dependencies. In the past, I worked hard to strictly limit the number of modules imported by ``test.libregrtest``.